src/corelibs/U2Algorithm/src/util_msa_consensus/MaConsensusAlgorithmSimpleExtended.cpp

```diff
@@ -171,11 +172,6 @@ char MaConsensusAlgorithmSimpleExtended::getConsensusChar(const MultipleAlignmen
             // There are no characters that fit the threshold, but we can merge a bit less popular characters
             return mergeCharacters(frequencies[frequency] << bestCharacter);
         }
-        if (frequencies[frequency].size() == 1) {
-            // There are no characters that fit the threshold and we found a single the most popular character.
-            // We need more characters to merge them with this one.
-            bestCharacter = frequencies[frequency].first();
-        }
```

Comment:
Restore this code, it is a specific case. See details in the comment to UGENE-5715.